aikau/src/main/resources/alfresco/services/CrudService.js

```diff
@@ -89,12 +89,15 @@ define(["dojo/_base/declare",
          });
 
          var noRefresh = lang.getObject("data.noRefresh", false, originalRequestConfig);
-         if (noRefresh === true) {
+         if (noRefresh === true) 
+         {
             // Don't make a refresh request...
-         } else {
+         } 
+         else 
+         {
             // When refreshing, check to see if a pubSubScope was provided...
-            var pubSubScope = lang.getObject("data.pubSubScope", false, originalRequestConfig) || "";
-            this.alfPublish(pubSubScope + "ALF_DOCLIST_RELOAD_DATA");
+            var pubSubScope = lang.getObject("responseScope", false, originalRequestConfig) || "";
+            this.alfPublish("ALF_DOCLIST_RELOAD_DATA", null, false, false, pubSubScope);
          }
```

Comment:
This all seems very verbose. The entire block could be rewritten as ...

```javascript
if (noRefresh !== true)
{
  this.alfPublish("ALF_DOCLIST_RELOAD_DATA", null, false, false, originalRequestConfig.alfResponseScope);
}
```

Of additional note, it would be good to use `topics.RELOAD_DATA_TOPIC`, and also shouldn't `alfResponseScope` be used instead of `responseScope`, as we should be taking the final response scope property from the payload, not the one passed into it?